test/unit/specs/hive/basic-store.js

```diff
@@ -0,0 +1,2237 @@
+import { expect } from 'chai'
+import { describe, it } from 'mocha'
+
+import Backendless, { APP_PATH, forTest, prepareMockRequest } from '../../helpers/sandbox'
+import { HiveTypes } from '../../../../src/hive/constants'
```

Comment:
I guess this is redundant, in the test we should check input/output only

```
import { HiveTypes } from '../../../../src/hive/constants'
import { HiveStore } from '../../../../src/hive/stores/base-store'
```

Reply:
done